draft-ietf-idr-link-bandwidth.xml

```diff
@@ -259,13 +259,35 @@
       the above preference.</t>
     </section>
 
+    <section anchor="History" title="Document History">
+      <t>The BGP Link Bandwidth Extended Community has evolved over several
+      versions of the IETF draft. In the earlier versions up to
+      draft-ietf-idr-link-bandwidth-08, only the non-transitive version of the
+      link bandwidth extended community was supported. However, starting from
+      draft-ietf-idr-link-bandwidth-09, both transitive and non-transitive
+      versions of the link bandwidth extended community are supported.</t>
+
+      <t>An old sender/receiver is a BGP speakers either use procedures upto
+      draft
+      (https://datatracker.ietf.org/doc/html/draft-ietf-idr-link-bandwidth-08)
+      or undocumented behavior for link bandwidth extended community.</t>
+
+      <t>A new sender/receiver is a BGP speaker that implements procedures
+      specified in this document. </t>
+
+      <t>Old receivers will not be able to interpret or utilize the new
+      procedures described from draft-ietf-idr-link-bandwidth-09, potentially
+      leading to inconsistencies in bandwidth management. The procedures
+      defined in this document does not cover old receivers.</t>
```

Comment:
<t> Receiving speaker needs to be upgraded to support the procedures defined in this document to provide full interop with both transitive and non-transitive versions of LBW. In order to keep changes to procedures simple, it is not a goal to provide interop between old Receiver and new Sender. </t>

Reply:
Removed the whole paragraph and added this suggested lines. Please check.